code/modules/mod/modules/_module.dm

```diff
@@ -245,8 +260,12 @@
 		used_overlay = overlay_state_inactive
 	else
 		return
+<<<<<<< HEAD
 	/* SKYRAT EDIT START - Making MODsuits mutant-compatible - ORIGINAL:
 	var/mutable_appearance/module_icon = mutable_appearance('icons/mob/mod.dmi', used_overlay, layer = standing.layer + 0.1)
+=======
+	var/mutable_appearance/module_icon = mutable_appearance('icons/mob/clothing/mod.dmi', used_overlay, layer = standing.layer + 0.1)
+>>>>>>> a61b884f302 (Time Eradication Modsuits (#63999))
```

Comment:
mfw the only conflict is entirely commented out anyways

Was just a change in filepath, from icons/mob/mod.dmi to icons/mob/clothing/mod.dmi

```suggestion
	/* SKYRAT EDIT START - Making MODsuits mutant-compatible - ORIGINAL:
	var/mutable_appearance/module_icon = mutable_appearance('icons/mob/clothing/mod.dmi', used_overlay, layer = standing.layer + 0.1)
```